webapp/lib/loader.dart

```diff
@@ -0,0 +1,16 @@
+/**
```

Comment:
Consider loader_ui.dart?

Reply:
Done, thanks, I'll send another PR shortly to rename the snackbar too for consistency.